game/die.h

```diff
@@ -0,0 +1,12 @@
+#include <stdlib.h>     /* srand, rand */
+#include <time.h> 
+
+// Class to represent a die of n sides.
+class Die {
+public:
+    Die(int n);
```

Comment:
Go ahead and just put the constructor here

Reply:
done